### PR TITLE
Update release.md with milestone step

### PR DESCRIPTION
**Pull request by awgreene**

### CODE REVIEW

File: doc/design/release.md [hunk 1]
```diff
@@ -1,14 +1,18 @@
 # Steps to create a new release
 
-## Step 0: Installing Requirements
+## Step 0: Review the Release Milestone
+If the release you plan to create corressponds with an existing [milestone](https://github.com/operator-framework/operator-lifecycle-manager/milestone/), make sure that all features have been committed. If a feature will not be added to the release be sure to remove it from the milestone.
```
Comment (benluddy) [inline doc/design/release.md:4]:
```suggestion
If the release you plan to create corresponds with an existing [milestone](https://github.com/operator-framework/operator-lifecycle-manager/milestone/), make sure that all features have been committed. If a feature will not be added to the release be sure to remove it from the milestone.
```

**Comment by ecordell** [comment]
/hold

Tests seem to have an issue, we should investigate on Monday

**Pull request merged**